Integrations/integration-MaxMind_GeoIP2.yml

```diff
@@ -0,0 +1,293 @@
+description: Enriches IP addresses
+detaileddescription: 'The MaxMind GeoIP2 integration allows you to query the MaxMind
+  API service and retrieve a JSON of all details.'
+configuration:
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Account ID
+  name: account
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Use system proxy
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Service Version
+  name: mode
```

Comment:
A thought about this param, shouldn't we make it as an optional argument while calling the command?
Also not sure if it's ok to expose such an option for an IP command, but if you agree lets check that out

Reply:
It was originally that way as that was my thought too, but speaking with @michalgold , we decided that setting the value as a param was more effective since we didn't want to create additional arguments for the !ip command.